frontend-react/playwright.config.ts

```diff
@@ -110,11 +103,4 @@ export default defineConfig<TestOptions>({
         //   use: { ...devices['iPhone 12'] },
         // },
     ],
-    webServer: {
-        command: `yarn cross-env VITE_IDLE_TIMEOUT=25000 yarn run preview:build:${isCi ? "ci" : "test"}`,
-        url: "http://localhost:4173",
-        timeout: 1000 * 180,
-        stdout: "pipe",
-        // reuseExistingServer: !process.env.CI,
-    },
 });
```

Comment:
When I try running `npm run test:e2e-ui` or `CI=true npm run test:e2e-ui`, I keep getting the following error within Playwright:

```
Error: page.goto: net::ERR_ABORTED; maybe frame was detached?
Call log:
  - navigating to "http://localhost:4173/login", waiting until "domcontentloaded"
```

Reply:
I'm assuming you meant `yarn run`? Try with the latest version and let me know if you still have issues.